pkg/controller/storagecluster/stork.go

```diff
@@ -24,6 +24,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
+
```

Comment:
nit: No need to create so many sections in the imports. Remove the empty lines and let the editor arrange the imports.

Reply:
ack